### PR TITLE
Addition of Rate Limit standardization headers

### PR DESCRIPTION
**Pull request by sanyamdogra**
For #979 
* [x] Updated `express-rate-limit` to 5.1.1
* [x] Added `express-slow-down` 
* [ ] Add support for `delayMs` and `delayAfter` to not break stuff.
* [ ] Finally, make it work! 

cc @ioggstream

### CODE REVIEW

File: lib/policies/rate-limit/rate-limit.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 const RateLimit = require('express-rate-limit');
+// const SlowDown = require('express-slow-down');
```
Comment (ioggstream) [inline lib/policies/rate-limit/rate-limit.js]: If we don't need this line, just remove it.

Comment (sanyamdogra) [reply lib/policies/rate-limit/rate-limit.js]: It's work in progress, we'll definitely need this line. `express-slow-down` contains the deprecated functions.

**Comment by ioggstream** [comment]
@sanyamdogra try to propose a couple of solutions to this problem. Take *some time* to write down your ideas and expose them to Nathan, which has a deep knowledge of the plugins.

**Comment by sanyamdogra** [comment]
> @sanyamdogra try to propose a couple of solutions to this problem. Take _some time_ to write down your ideas and expose them to Nathan, which has a deep knowledge of the plugins.

Yes, let me do some research and discuss it with him. Thanks!

**Pull request closed without merging**